docs/data-ai/train/annotate-images.md

```diff
@@ -0,0 +1,472 @@
+---
+linkTitle: "Annotate images for training"
+title: "Annotate images for training"
+weight: 40
+layout: "docs"
+type: "docs"
+description: "Annotate images to train a machine learning model."
+---
+
+You can either manually add annotations through the Viam web UI, or add annotations with an existing ML model.
+
+## Prerequisites
+
+{{% expand "A machine connected to Viam" %}}
+
+{{% snippet "setup.md" %}}
+
+{{% /expand %}}
+
+{{% expand "A camera, connected to your machine, to capture images" %}}
+
+Follow the guide to configure a [webcam](/operate/reference/components/camera/webcam/) or similar [camera component](/operate/reference/components/camera/).
+
+{{% /expand %}}
+
+## Classify images with tags
+
+Classification determines a descriptive tag or set of tags for an image.
```

Comment:
an image here might be great and more immediatly convey this but that doesn't need to necessarily happen with this PR

Reply:
I'll brainstorm some ideas for this and submit a request